common/src/main/java/org/opensearch/ml/common/input/nlp/TextDocsMLInput.java

```diff
@@ -124,6 +129,14 @@ public TextDocsMLInput(XContentParser parser, FunctionName functionName) throws
                 case RESULT_FILTER_FIELD:
                     resultFilter = ModelResultFilter.parse(parser);
                     break;
+                case ML_PARAMETERS_FIELD:
```

Comment:
Can we add a test for this case?

Reply:
This test implements a serialization round-trip. Don't you think this is covered?
https://github.com/opensearch-project/ml-commons/pull/2123/files#diff-a4fe83870d62cd423be730edbf6fa08b8d36ca848dcc32b8c9de6b56f2eac286R72

Reply:
I don't think this file is even being used anywhere. So you can skip comment for this file. I'll plan to remove this file later.

Reply:
Oh, okay. I'll revert my changes then.